core/src/main/java/com/linecorp/armeria/client/retry/RetryingRpcClient.java

```diff
@@ -109,7 +109,7 @@ private void doExecute0(ClientRequestContext ctx, RpcRequest req,
 
         final int totalAttempts = getTotalAttempts(ctx);
         if (totalAttempts > 1) {
-            derivedCtx.addAdditionalRequestHeader(TOTAL_ATTEMPTS, Integer.toString(totalAttempts));
+            derivedCtx.addAdditionalRequestHeader(ARMERIA_RETRY_COUNT, Integer.toString(totalAttempts - 1));
```

Comment:
Could use `setAdditionalRequestHeader()` instead of `addAdditionalRequestHeader()`?